dot/network/service.go

```diff
@@ -87,6 +89,11 @@ var (
 		Name:      "outbound_total",
 		Help:      "total number of outbound streams",
 	})
+	processStartTimeGauge = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "substrate",
+		Name:      "process_start_time_seconds",
+		Help:      "gossamer process start time seconds, using substrate namespace so zombienet detects node start",
```

Comment:
```suggestion
		Help:      "gossamer process start seconds unix timestamp, using substrate namespace so zombienet detects node start",
```

just plainly saying it's a unix timestamp, since it did confuse me a bit (thought it was seconds elapsed until gossamer was ready)